attributes/default.rb

```diff
@@ -338,7 +338,8 @@
 )
 default['cfncluster']['lustre']['base_url'] = value_for_platform(
   'centos' => {
-    '>=8' => "https://fsx-lustre-client-repo.s3.amazonaws.com/el/8/x86_64/",
+    # Get architecture if centos8, node['kernel']['machine'] should be 'x86_64' or 'aarch64'
```

Comment:
minor: I'd rewrite the comment to be more compact: 
`# node['kernel']['machine'] contains the architecture: 'x86_64' or 'aarch64'`